packages/plugin-core/src/commands/pods/MarkdownExportPodCommand.ts

```diff
@@ -0,0 +1,200 @@
+  RunnableMarkdownV2PodConfig,
+  string
+> {
+  public key = "dendron.markdownexportv2";
+
+  public async gatherInputs(
+    opts?: Partial<MarkdownV2PodConfig>
+  ): Promise<RunnableMarkdownV2PodConfig | undefined> {
+    if (isRunnableMarkdownV2PodConfig(opts)) {
+      return opts;
+    }
+
+    // First get the export scope:
+    const exportScope =
+      opts && opts.exportScope
+        ? opts.exportScope
+        : await PodUIControls.promptForExportScope();
+
+    if (!exportScope) {
+      return;
+    }
+
+    const destination = await this.promptUserForDestination();
+    if (!destination) {
+      return;
+    }
+
+    const wikiLinkToURL = await this.promptUserForWikilinkToURLSetting();
+    if (wikiLinkToURL === undefined) {
+      return;
+    }
+
+    const config = {
+      exportScope,
+      wikiLinkToURL,
+      destination,
+    };
+
+    // If this is not an already saved pod config, then prompt user whether they
+    // want to save as a new config or just run it one-time
+    if (!opts?.podId) {
+      const choice = await PodUIControls.promptToSaveInputChoicesAsNewConfig();
+
+      if (choice !== undefined && choice !== false) {
+        const configPath = ConfigFileUtils.genConfigFileV2({
+          fPath: path.join(
+            getExtension().podsDir,
+            "custom",
+            `config.${choice}.yml`
+          ),
+          configSchema: MarkdownExportPodV2.config(),
+          setProperties: _.merge(config, {
+            podId: choice,
+            podType: PodV2Types.MarkdownExportV2,
+          }),
+        });
+
+        vscode.window.showInformationMessage(
+          `Configuration saved to ${configPath}`
+        );
+      }
+    }
+
+    return config;
+  }
+
+  public onExportComplete({
+    exportReturnValue,
+    payload,
+    config,
+  }: {
+    exportReturnValue: string;
+    payload: string | NoteProps;
+    config: RunnableMarkdownV2PodConfig;
+  }): void {
+    if (config.destination === "clipboard") {
+      if (typeof payload === "string") {
+        vscode.env.clipboard.writeText(exportReturnValue);
+      } else {
+        //TODO: This error needs to be thrown earlier.
+        // throw new Error(
+        //   "Cannot have clipboard be the destination on a multi-note export"
+        // );
+
+        vscode.env.clipboard.writeText(exportReturnValue);
+      }
+    } else {
+      throw new Error("Not yet implemented");
+    }
+
+    vscode.window.showInformationMessage(
+      "Finished running Markdown export pod."
+    );
+  }
+
+  public createPod(config: RunnableMarkdownV2PodConfig): ExportPodV2<string> {
+    return new MarkdownExportPodV2({
+      config,
+      engine: getEngine(),
+      dendronConfig: getDWorkspace().config,
+    });
+  }
+
+  /*
+   * Region: UI Controls
+   */
+
+  /**
+   * Prompt user with simple quick pick on which wikilink conversion setting to
+   * use
+   * @returns
+   */
+  private async promptUserForWikilinkToURLSetting(): Promise<
```

Comment:
since we store the pod config as JSOONSchema, we should be able to have a factory that translates from JSONSchema -> vscode.quickpick. not necessary now but good future work. we could use another factory to translate JSONSchema into a form for a webview/UI based appraoch

Reply:
This is a cool idea - I think it should work for most configs.  There are some fields that we may build special UI's for - for example, file pickers, google doc files, etc.